app/src/components/Credits.tsx

```diff
@@ -1,40 +1,18 @@
-import Image from 'next/image'
 import { FC } from 'react'
 
 const Credits: FC = () => {
   return (
-    <div className="credits mt-6 flex flex-row items-center justify-center text-xs text-turtle-level5 sm:text-sm">
+    <div className="credits mt-6 text-center text-xs text-turtle-level5">
       Made with love by{' '}
-      <a
-        href="https://www.velocitylabs.org"
-        target="_blank"
-        rel="noopener noreferrer"
-        className="px-2"
-      >
-        <Image
-          src={'/velocitylabs.svg'}
-          alt={'Velocity Labs'}
-          width={24}
-          height={24}
-          className="rounded-full border-1"
-        />
-      </a>
-      {' ・ '}
-      Powered by{' '}
-      <a
-        href="https://docs.snowbridge.network/"
-        target="_blank"
-        rel="noopener noreferrer"
-        className="px-2"
-      >
-        <Image
-          src={'/snowbridge.svg'}
-          alt={'Snowbridge Network'}
-          width={24}
-          height={24}
-          className="rounded-full border-1"
-        />
+      <a href="https://www.velocitylabs.org" target="_blank" rel="noopener noreferrer">
+        Velocity Labs
       </a>
+      <div className="mt-2">
+        Powered by{' '}
+        <a href="https://docs.snowbridge.network/" target="_blank" rel="noopener noreferrer">
+          Snowbridge Network
```

Comment:
Snowbridge is enough 👍 

```suggestion
          Snowbridge
```